TmpFilesCheck.py

```diff
@@ -37,11 +38,16 @@ def check(self, pkg):
             if not stat.S_ISREG(pkgfile.mode):
                 printWarning(pkg, "tmpfile-not-regular-file", fn)
                 continue
+            basename = os.path.basename(fn)
 
             pattern = re.compile(
                 r'systemd-tmpfiles --create .*%s' % re.escape(fn))
-            if (not postin or not pattern.search(postin)) and \
-                    (not prein or not pattern.search(prein)):
+            pattern_basename = re.compile(
+                r'systemd-tmpfiles --create .*%s' % re.escape(basename))
```

Comment:
is there a case where "pattern" would not also match "pattern_basename" ? I can't think of any atm. hence we could simplify the code to just check for pattern_basename only and always.

Reply:
hmm, indeed, pattern_basename would cover both cases, so I'll leave only that (and use ```pattern 
``` for the variable name which I like better)